.github/CODEOWNERS

```
@@ -0,0 +1 @@
+smoltcp-rs astro
```

Shouldn't that be:
```suggestion
* @smoltcp-rs
```
instead?

---

Doesn't it have to be in `@org/team` syntax?

---

Not sure, but as is it's definitely wrong. ;)